shell/app/node_main.cc

```diff
@@ -62,40 +62,21 @@ int SetNodeCliFlags() {
       "--use-openssl-ca",
   });
 
-  const auto argv = base::CommandLine::ForCurrentProcess()->argv();
   std::vector<std::string> args;
-
-  // TODO(codebytere): We need to set the first entry in args to the
-  // process name owing to src/node_options-inl.h#L286-L290 but this is
-  // redundant and so should be refactored upstream.
   args.reserve(argv.size() + 1);
-  args.emplace_back("electron");
 
   for (const auto& arg : argv) {
-#if BUILDFLAG(IS_WIN)
-    const auto& option = base::WideToUTF8(arg);
-#else
-    const auto& option = arg;
-#endif
-    const auto stripped = base::StringPiece(option).substr(0, option.find('='));
-    if (disallowed.contains(stripped)) {
-      LOG(ERROR) << "The Node.js cli flag " << stripped
+    const auto key = base::StringPiece(arg).substr(0, arg.find('='));
```

Comment:
```suggestion
    const auto key = arg.substr(0, arg.find('='));
```

Reply:
@zcbenz that's terser, but does cause a new `std::string` to be constructed with a copy of the front of `arg`.

Probably (very slightly) better to use the string view instead?

Reply:
Didn't realize that, I'm good as it is.